code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -608,6 +608,19 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 		return amount_cache[reagent] >= max(0,amount)
 	return 0
 
+/datum/reagents/proc/has_any_reagents(var/list/input_reagents, var/amount = -1)		//returns true if any of the input reagents are found
+	var/has = FALSE
```

Comment:
You can just use the `.` variable which is built in, so you can remove the `return has`